Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM golang:1.6.3
+
+COPY . /go/src/github.com/cloudflare/redoctober
+RUN go get github.com/cloudflare/redoctober/...
```

Comment:
Alternatively, it might make sense to pin a version, but perhaps not if the goal is to do development.